doc/en/Common.md

```diff
@@ -0,0 +1,101 @@
+# Overview
+
+Alchemist is an tool for assisting component-oriented FPGA development.
+It builds a digital circuit combining circuit components which are written in software programming language (i.e. C-language).
+The components are synthesized into HDL (Hardware Description Language) and synthezied, placed and routed onto FPGA using FPGA tools.
+
+# Environment
+
+- OS: Ubuntu 18.04 LTS
+- Python 3.7>=
+- FPGA tools: Vivado, Vivado HLS (2019.2)
+- Fast RTPS (v1.7.2)
+
+# Installation
+
+## Fast RTPS
+
+### Dependency
+
+- g++, make
+- CMake
+- Git
+- JDK8
+- Gradle
+
+```
+$ sudo apt install build-essential cmake git openjdk-8-jdk gradle -y
+$ sudo apt install libncurses5 -y
+```
+
+### Fast-RTPS compilation and install
+
+```
+
+$ git clone https://github.com/eProsima/Fast-RTPS
+$ cd Fast-RTPS
+$ git checkout -b v1.7.2 remotes/origin/release/1.7.2
+$ mkdir bulid
+$ cd build
+$ cmake -DTHIRDPARTY=ON -DBUILD_JAVA=ON ..
+$ make -j8
+$ sudo make install
+```
+
+## Vivado, Vivado HLS
+
+Refer to [Vivado Design Suite User Guide UG973(v2019.1)](https://www.xilinx.com/support/documentation/sw_manuals/xilinx2019_1/ug973-vivado-release-notes-install-license.pdf).
+
+## alchemist_py
+
+```
+$ git clone https://github.com/ohkawatks/alchemist_py
```

Comment:
The URL is your repository.
Do you mind to fix it later?

Reply:
No, I don't. Please fix it to the correct one.

Reply:
F.Y.I.) I made a clean Ubuntu18.04 environment and confirmed the following commands works:

```
ohkawa@alchemist:~$ sudo apt install python3-pip
ohkawa@alchemist:~$ sudo pip3 install git+https://github.com/Kenta11/alchemist_py
```